extension/src/experiments/webview/messages.ts

```diff
@@ -170,7 +170,6 @@ export class WebviewMessages {
 
   private setExperimentStars(ids: string[]) {
     this.experiments.toggleStars(ids)
-    this.sendWebviewMessage()
```

Comment:
`this.notifyChanged()` below already sends the data. We have to send the data to the webview and update the trees because we added the filter to starred action.